recipe/0103-poetry-reading-annotations/manifest2.json

```diff
@@ -0,0 +1,48 @@
+{
+    "@context": "http://iiif.io/api/presentation/3/context.json",
+    "id": "{{ id.url }}",
+    "type": "Manifest",
+    "label": { "en": [ "Anne Sexton, Poetry Reading, 1974 -- Her Kind" ] },
+    "items": [
+      {
+        "id": "{{ id.path }}/canvas/1",
+        "type": "Canvas",
+        "duration": 107,
+        "items": [
+          {
+            "id": "{{ id.path }}/canvas/1/paintings",
+            "type": "AnnotationPage",
+            "items": [
+              {
+                "id": "{{ id.path }}/canvas/1/painting/1",
+                "type": "Annotation",
+                "motivation": "painting",
+                "body": {
+                  "id": "https://library.harvard.edu/poetry/audio/listeningbooth//PS3537E915A6x1974/Her_Kind.mp3",
```

Comment:
// in URL

Reply:
This is now fixed with Sara's commit: https://github.com/IIIF/cookbook-recipes/pull/130/commits/e3e42c7bb422c65595e2fcfd5ca51a04f873cf98